core/schema/src/main/liquibase/19.1.0/changelog.xml

```diff
@@ -142,4 +142,331 @@
       delete from monitoringlocations where id='';
     </sql>
   </changeSet>
+
+  <!-- NMS-6584: Upgrade to Quartz 2 schema -->
+  <!-- http://www.quartz-scheduler.org/documentation/quartz-2.x/migration-guide.html -->
+  <changeSet author="seth" id="19.1.0-upgrade-to-quartz-2-schema">
+
+    <!-- Drop tables that are no longer used -->
+
+    <dropTable tableName="qrtz_job_listeners"/>
+    <dropTable tableName="qrtz_trigger_listeners"/>
+
```

Comment:
We never used "job listeners" or "trigger listeners" in Quartz 1 and whatever they are, they no longer exist in Quartz 2. We only store jobs and cron triggers for database reports in the DB afaik.